config/endpoint.go

```diff
@@ -37,12 +37,17 @@ func NewEndpointFromCloneURLWithConfig(url string, c *Configuration) Endpoint {
 		return e
 	}
 
+	if strings.HasSuffix(url, "/") {
```

Comment:
We could, but it does a lot more stuff besides just chomping the trailing `/` that we don't need. These urls either come from the git remote config (`remote.origin.url`), or from custom lfs config (`lfs.url`, `remote.origin.lfsurl`).